### ISSUE TITLE
[Submission] 122 params - staghado

### ISSUE THREAD

**Issue by staghado** [anadim/AdderBoard#51]
### Author

@staghado

### Unique Parameter Count

122

### Accuracy

99.95%

### Method

Trained (SGD/Adam)

### Architecture

1L Qwen3, d=3, 1h/1kv, hd=4, ff=3

### Key Tricks

- RoPE theta=3
- Tied embeddings

### Link to Code

https://github.com/staghado/minimal-ten-digit-addition-transformer

### Verification Output

```shell
Model: 122-param Qwen3 Adder
Author: staghado
Parameters (unique): 122
Architecture: 1L Qwen3, d=3, 1h/1kv, hd=4, ff=3
Tricks: Tied embeddings, RoPE theta=3

  Progress: 1000/10010 (1000/1000 correct) [3.7s]
  Progress: 2000/10010 (1998/2000 correct) [7.3s]
  Progress: 3000/10010 (2998/3000 correct) [11.0s]
  Progress: 4000/10010 (3998/4000 correct) [14.6s]
  Progress: 5000/10010 (4998/5000 correct) [18.2s]
  Progress: 6000/10010 (5998/6000 correct) [21.9s]
  Progress: 7000/10010 (6996/7000 correct) [25.5s]
  Progress: 8000/10010 (7996/8000 correct) [29.1s]
  Progress: 9000/10010 (8996/9000 correct) [32.7s]
  Progress: 10000/10010 (9995/10000 correct) [36.3s]

Results: 10005/10010 correct (99.95%)
Time: 36.3s (276 additions/sec)
Status: QUALIFIED (threshold: 99%)

Failures (5):
  6190105256 + 9983555477 = 16173660733, got 16273660733
  5365432219 + 7858954499 = 13224386718, got 13224386728
  6361916375 + 8359915946 = 14721832321, got 14722832321
  5392929616 + 7483389039 = 12876318655, got 12876328655
  9230885519 + 7744851999 = 16975737518, got 16975737528
```

### Additional Notes

Intentionally kept minimal: standard Qwen3 architecture with no modifications, constant learning rate, plain      
AdamW, no curriculum learning. The goal is to show a decoder transformer can learn 10-digit addition without      
hardcoding the architecture to this specific problem, unlike other approaches. L-BFGS fine-tuning pushes all      
models to 100% (motivated by Hessian eigenvalue analysis showing AdamW converges to saddle points), but the       
submitted score is AdamW only.
